cookbook/security/custom_provider.rst

```diff
@@ -9,7 +9,10 @@ When a user submits a username and password, the authentication layer asks
 the configured user provider to return a user object for a given username.
 Symfony then checks whether the password of this user is correct and generates
 a security token so the user stays authenticated during the current session.
-Out of the box, Symfony has an "in_memory" and an "entity" user provider.
+Out of the box, Symfony has user providers ``in_memory`` and ``entity``.
+Additionally, you can use ``chain`` user provider to combine different 
+providers in your application.
```

Comment:
I like your proposal. It sounds very clear. Unfortunately I didn't feel my sentence as odd because I'm not a native speaker.